js/modules/Kanban/Kanban.js

```diff
@@ -1040,7 +1040,17 @@ class GLPIKanbanRights {
             const modal = $('#kanban-modal');
             modal.removeData();
             modal.data(data);
+            // Extract script elements from content to be manually inserted later with createElement to ensure they are executed
+            // Issue is noticed when content is injected multiple times. Scripts execute the first time only.
+            const scripts = $(content).find('script');
+            scripts.detach();
             modal.find('.modal-body').html(content);
+            scripts.each(function() {
+                const script = document.createElement('script');
+                script.type = 'text/javascript';
+                script.text = this.innerHTML;
+                modal.find('.modal-body').append(script);
+            });
```

Comment:
This sounds like a hack. What is the initial issue to solve with this precisely?

Reply:
The new team member modal content contains an inline script for the select2 dropdowns. The first time the modal is shown, the inline script is executed as expected. Subsequent injections of the content with the script results in the script not being evaluated again. Maybe there is some other way to fix the issue, but at the time this was the only thing I found.